src/js/select2/dropdown/tabs.js

```diff
@@ -106,11 +107,22 @@ define([
 
     // filter out the results for this specific tab
     $.each(params.data.results || [], function(k, res){
-      if (res.tabId === tabId) {
+      // if user is searching, tabs are hidden and results are displayed in one box
+      if (isSearchActive && !res.isSpecial) {
+        results.push(res);
+      } else if (!isSearchActive && res.tabId === tabId) {
         results.push(res);
       }
     });
 
+    if (isSearchActive && !this.isHidden) {
+      this.$tabsContainer.hide();
+      this.isHidden = true;
+    } else if (!isSearchActive && this.isHidden) {
+      this.$tabsContainer.show();
+      this.isHidden = false;
+    }
```

Comment:
ne vidim smisla v tem `isHidden`. 
To vse bi lahko napisal tako:
```
this.$tabsContainer.toggle(!isSearchActive);
```

Reply:
Pa res dela ta `toggle`. Nebi se meni to posvetilo. Niti nisem vedel da `toggle` sprejema `boolean`.